KamerConnect.View.MAUI/UpdateAccount.xaml.cs

```diff
@@ -9,26 +10,34 @@ namespace KamerConnect.View.MAUI
     public partial class UpdateAccount : ContentPage
     {
         private FileService _fileService;
-
-        public UpdateAccount(FileService fileService)
+        private PersonService _personService;
+        private Person _currentPerson;
+        
+        public UpdateAccount(FileService fileService, PersonService personService, Person currentPerson)
```

Comment:
Je geeft nu een Person mee in de page, maar Niek heeft waarschijnlijk een functie gemaakt waardoor je deze uit de sessie kan halen. Als het goed is is dat van Niek af dus kan je die gebruiken.